### PR TITLE
Add Symfony conferences 2023

### PR DESCRIPTION
**Pull request by GromNaN**
- CFP for SymfonyLive Paris ended
- SymfonyCon is announced in Brussels, CFP open
- SymfonyOnline is scheduled

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -126,7 +126,7 @@ All the events (past and coming) are available publicly in JSON: https://develop
 * 23: [Apprise Conf](https://appriseconf.com/) - Oslo (Norway) <a href="https://docs.google.com/forms/d/e/1FAIpQLSeL3DOqPTM8GgaJONtPI-LmzCZCCU-SIxu1hepAhdQxfpeM-g/viewform"><img alt="CFP Apprise Conf" src="https://img.shields.io/static/v1?label=CFP&message=until%31-January-2023&color=green"></a>
 * 23: [DevDay](https://www.devday.be/) - Mons (Belgium) <a href="https://sessionize.com/devday-2023/?e=1688a2"><img alt="CFP DevDay" src="https://img.shields.io/static/v1?label=CFP&message=from%20October-24%20to%20January-31-2023&color=green"></a>
 * 23-24: [Wasm I/O](https://wasmio.tech) - Barcelona (Spain) <a href="https://www.papercall.io/wasmio23"><img alt="CFP Wasm I/O" src="https://img.shields.io/static/v1?label=CFP&message=until%2017-January-2023&color=green"></a>
-* 23-24: [SymfonyLive Paris](https://live.symfony.com/2023-paris/) - Paris (France) <a href="https://live.symfony.com/2023-paris/cfp"><img alt="CFP SymfonyLive Paris" src="https://img.shields.io/static/v1?label=CFP&message=until%2016-December-2022&color=yellow<"></a>
+* 23-24: [SymfonyLive Paris](https://live.symfony.com/2023-paris/) - Paris (France)
```
Comment (scraly) [inline README.md:129]:
```suggestion
* 23-24: [SymfonyLive Paris](https://live.symfony.com/2023-paris/) - Paris (France) <a href="https://live.symfony.com/2023-paris/cfp"><img alt="CFP SymfonyLive Paris" src="https://img.shields.io/static/v1?label=CFP&message=until%2016-December-2022&color=red<"></a>
```

**Review by scraly — CHANGES_REQUESTED** [review]
thanks
about closed cfp, just put in red the cfp instead of delete it please :)

**Comment by GromNaN** [comment]
Ok, that's much better.

**Comment by scraly** [comment]
merci :)

**Pull request merged**